service/src/main/resources/application.yml

```diff
@@ -21,6 +17,8 @@ spring:
       minimum-idle: 8
       idle-timeout: 10000
       connectionTestQuery: "SELECT 1"
+    dbcp2:
```

Comment:
Please just remove the dbcp2 section.  We aren't using that driver and it isn't used.  I mistakenly left it in.